src/Components/Forms/DiagnosisAndRecommendations/RecommendationsOnRelease /index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Title from 'Assets/Elements/Title';
+import CustomizedTextField from 'Assets/Elements/CustomizedTextField';
+import { Grid } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { StyledFormGroup } from 'Assets/Elements/StyledFormGroup';
+import { StyledButton } from 'Assets/Elements/StyledButton';
+import { StyledDivider } from '../Style';
+import { useFormContext } from 'react-hook-form';
+const RecommendationsOnRelease = () => {
+  const { t } = useTranslation();
+  const {
+    permission,
+    register,
+    watch,
+    setPopUpProps,
+    setValue,
+  } = useFormContext();
+  const instructionsForFurtherTreatment = watch(
+    'instructionsForFurtherTreatment',
+  );
+  const callBack = (data, name) => {
+    setValue(name, data);
+  };
+
+  const handlePopUpProps = (title, fields, id, callBack, name) => {
+    setPopUpProps((prevState) => {
+      return {
+        ...prevState,
+        popupOpen: true,
+        formFieldsTitle: title,
+        formFields: fields,
+        formID: id,
+        setTemplatesTextReturned: callBack,
+        name,
+      };
+    });
+  };
+  return (
+    <StyledFormGroup>
+      <Title
+        label={t('Recommendations on release')}
+        fontSize='22px'
+        bold
+        color='#000b40'
+      />
+      <StyledDivider />
+      <Grid
+        container
+        direction='row'
+        justify='flex-start'
+        alignItems='baseline'>
+        <CustomizedTextField
+          name='instructionsForFurtherTreatment'
+          inputRef={register}
+          label={t('Instructions for further treatment')}
+          width='45%'
+          multiline
+          InputLabelProps={{
+            shrink: instructionsForFurtherTreatment ? true : false,
+          }}
+          disabled={permission === 'view' ? true : false}
+        />
+        <StyledButton
```

Comment:
replace `StyledButton`  with new component in the asset for "select template" - `StyledSelectTempleteButton`
all the buttons of select template contain the same style and same title 
why we need to copy
```
  width='113px'
          height='32px'
          color='primary'
          margin='0 16px'
          variant='outlined'
          size='small'
```
In all the buttons

Reply:
Ok